MonitorVoos/Main/forms.py

```diff
@@ -42,27 +43,26 @@ class Meta:
         ]
 
 
-class Newflightform(forms.Form):
+class Newflightform(ModelForm):
```

Comment:
Aqui no “form” você não precisa sobrescrever os campos que já foram especificados no model, apenas caso queira adicionar um novo é necessário. Nos fields, os nomes devem estar igual no model, departure e arrival airport estão diferentes que o especificado no model. 

https://stackoverflow.com/questions/21164798/django-modelform-with-foreign-key

Modificando para o jeito que tá no exemplo, vai ter um problema que o dropdown com as opções vão aparcer como objeto de tal coisa.
![image](https://user-images.githubusercontent.com/51101577/200386071-22e30912-3b63-444e-b636-70de6a93b897.png)

Pra contornar isso tem que criar um def __init__ nos models:
https://stackoverflow.com/questions/65184364/django-modelform-display-model-field-of-foreign-key-instead-object-id

Aí um exemplo de como ficaria é assim, nome do piloto - anac code
![image](https://user-images.githubusercontent.com/51101577/200386326-b93b15ea-5f3b-49e9-8eab-2d0e581b0334.png)